root/static/scripts/jquery.flot.musicbrainz_events.js

```diff
@@ -74,4 +74,4 @@
         name: 'musicbrainzEvents',
         version: '1.0'
     });
-})(jQuery);
+}(jQuery));
```

Comment:
Actually using default `outside` option for `wrap-life` rule, but current code follows `inside` style.

Reply:
I checked, and there's four fixes needed for inside vs two for outside, so this is still better it seems :)

Reply:
I've always used `}(...))`, mostly as [a habit from JSLint](https://youtu.be/taaEzHI9xyY?t=2019).

But note that for arrow functions, `((x) => { return x }(1))` is actually a syntax error; you have to write it like `((x) => { return x })(1)`. This rule doesn't apply to arrow functions, but it's interesting to note and maybe an argument for the other way if that bothers you. On the other hand, our usage of these should be extremely rare, if we ever do use them at all.

Reply:
I dropped the wrap-iife commit to hopefully unblock this.